package.json

```diff
@@ -1,41 +1,47 @@
 {
-  "name": "math-nodes",
-  "version": "0.1.10",
-  "description": "Creation and identification utility functions for nodes in math-ast ASTs",
-  "main": "dist/math-nodes.js",
-  "scripts": {
-    "lint": "eslint lib",
-    "prepublish": "webpack",
-    "test": "jest"
-  },
-  "pre-commit": "lint",
-  "jest": {
-    "setupTestFrameworkScriptFile": "<rootDir>/node_modules/babel-polyfill/dist/polyfill.js",
-    "transform": {
-      "^.+\\.jsx?$": "babel-jest"
+    "name": "math-nodes",
+    "version": "0.1.10",
+    "description": "Creation and identification utility functions for nodes in math-ast ASTs",
+    "main": "dist/math-nodes.js",
+    "scripts": {
+        "lint": "eslint lib",
+        "prepublish": "webpack",
+        "test": "jest"
+    },
+    "pre-commit": "lint",
+    "jest": {
+        "setupTestFrameworkScriptFile": "<rootDir>/node_modules/babel-polyfill/dist/polyfill.js",
+        "transform": {
+            "^.+\\.jsx?$": "babel-jest"
+        }
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/semantic-math/math-nodes.git"
+    },
+    "author": "Kevin Barabash <kevinb7@gmail.com>",
+    "license": "MIT",
+    "bugs": {
+        "url": "https://github.com/semantic-math/math-nodes/issues"
+    },
+    "homepage": "https://github.com/semantic-math/math-nodes#readme",
+    "devDependencies": {
+        "babel-core": "^6.22.1",
+        "babel-eslint": "^7.1.1",
+        "babel-loader": "^6.2.10",
+        "babel-plugin-transform-flow-strip-types": "^6.22.0",
+        "babel-plugin-transform-object-rest-spread": "^6.22.0",
+        "babel-polyfill": "^6.23.0",
+        "babel-preset-es2015": "^6.22.0",
+        "eslint": "^3.15.0",
+        "eslint-plugin-flowtype": "^2.30.0",
+        "flow-bin": "^0.38.0",
+        "jest": "^20.0.3",
+        "json-stable-stringify": "^1.0.1",
+        "pre-commit": "^1.2.2",
+        "webpack": "^2.2.1"
+    },
+    "dependencies": {
+        "math-parser": "^0.10.5"
```

Comment:
Please revert.  We're already using `jest` so you should be able to use `.toMatchSnapshot()`.  My understanding is that that you can only have one call to `.toMatchSnapshot()` per `it` spec.